README.md

```diff
@@ -229,8 +229,20 @@ The `./.actions/deploy-system` action should only be run after a job running the
       - uses: gridpoint-com/actions-nerves-system@v1
       - name: Deploy nerves_system
         uses: ./.actions/deploy-system
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+> [!NOTE]
+> `GITHUB_TOKEN` is provided in every action as a secret which needs
+> to be passed to the job using it via `${{ secrets.GITHUB_TOKEN }}
```

Comment:
missing backtick at the end of this line

Reply:
Derp 🤦🏼 Fixed!